### ISSUE TITLE
Make `julia-version` optional?

### ISSUE THREAD

**Issue by davidanthoff** [julia-actions/install-juliaup#60]
I think that could be optional, with a default value of `release`?

**Comment by DilumAluthge** [comment]
That seems reasonable. Here's one question though: what happens if/when a Julia 2.0 is released (in the distant future, if it ever does happen)? If we default to `release`, then users will automatically have their workflows start running on Julia 2.0, without any change on their end. Could that be problematic?

Would it make sense to instead default to the `1` channel? And then, if/when Julia 2.0 is released, we would make a breaking release of this action, in which we changed the default value in `action.yml` from `1` to `2`? And then, users of this action won't automatically get upgraded to Julia 2.0 until they merge the Dependabot PR to upgrade to the new breaking release of this action?

**Comment by davidanthoff** [comment]
Yeah, it is a trade-off... I think I'm leaning towards either making it mandatory (i.e. leave as is) or making the default `release`, i.e. your on your own... I think I'm not a huge fan of making `1` the default, don't really have a good reason, but somehow I would then like a mandatory input more ;) I'll post this on the slack channel, maybe others have thoughts.

**Comment by DilumAluthge** [comment]
We didn't get any further feedback from Slack.

Between keeping it mandatory (leaving the current behavior in place), and defaulting to `release`, I think I'd prefer we keep it mandatory. I don't love the idea of someone's workflow suddenly switching from Julia 1.x to Julia 2.x without any manual intervention on their part.
